src/components/InvitationsTable/InvitationsTable.tsx

```diff
@@ -0,0 +1,65 @@
+import { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useParams } from 'react-router-dom';
+import { format } from 'date-fns';
+
+import { users } from 'redux/modules';
+import { useAppDispatch } from 'redux/store';
+import { Row, Table } from 'components/Table';
+import { getHeadCells } from './InvitationsTable.const';
+import { StyledTitle } from './Invitationstable.styles';
+import { Invitation } from './InvitationsTable.types';
+
+export const InvitationsTable = () => {
+  const { t } = useTranslation('app');
+  const { id } = useParams();
+  const dispatch = useAppDispatch();
+
+  const [rows, setRows] = useState<Row[]>([]);
```

Comment:
you need to store rows in redux, because I will call this thunk from Add user component